### ISSUE TITLE
Package Does Not Automatically Load With ELPA

### ISSUE THREAD

**Issue by Sammidysam** [hadronzoo/theme-changer#6]
After installing via ELPA, I can add the following code to my `.emacs`.

```elisp
(eval-after-load "theme-changer" '(change-theme 'solarized-light 'solarized-dark))
```

But the code is not run on start-up; I have to manually load the file `theme-changer.el` or `theme-changer.elc`, and then the code will be run.  Therefore, the package is not automatically loaded when installed via ELPA, which is a problem.

**Comment by shackra** [comment]
is `(package-initialize)` at the top of your emacs configuration?

**Comment by Sammidysam** [comment]
It is not necessarily at the top, but it is still run near the beginning of the Emacs configuration process.  I think in my previous configuration that I was using when I opened the issue, I didn't run `(package-initialize)`, but I am not sure.  I just checked though, and this problem still exists, even with having `(package-initialize)` run.

**Comment by shackra** [comment]
Ran... but at the beginning of your Emacs configuration? Take a look on my Emacs configuration https://github.com/shackra/.emacs.d/blob/master/init.el

El viernes 25 de diciembre del 2015 a las 2159 horas, Sam Craig escribió:

> It is not necessarily at the top, but it is still run near the beginning of the Emacs configuration process.  I think in my previous configuration that I was using when I opened the issue, I didn't run `(package-initialize)`, but I am not sure.  I just checked though, and this problem still exists, even with having `(package-initialize)` run.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hadronzoo/theme-changer/issues/6#issuecomment-167275553


-- 
👋 Pax et bonum.
Jorge Araya Navarro
https://es.gravatar.com/shackra

**Comment by Sammidysam** [comment]
I was going through my Emacs configuration, and I did not see `eval-after-load` anywhere.  I remember using that command in the past, but it might have broken with the current structure of my configuration.  I also see that in the README for this package, never does it say to use `eval-after-load`.  Finally, I made this issue over a year ago, so I really do not remember my train of thought about it.  As a result of these problems, I'll close the issue.  I don't think it is actually prevalent.

I think autoloading has to be set up by the package, and here it isn't, so it's not a real issue.  This was probably more my mistake for thinking every package had autoloading.
